pkg/configure/oneagent/pmc/ruxit/types.go

```diff
@@ -20,9 +20,33 @@ type Property struct {
 	Value   string `json:"value"`
 }
 
-func (pmc ProcConf) ToMap() ProcMap {
+// ToString creates the content of the configuration file, the sections and properties are printed in a sorted order, so it can be tested.
+func (pc ProcConf) ToString() string {
+	if pc.InstallPath != nil {
+		pm := pc.ToMap()
+		pm = pm.SetupReadonly(*pc.InstallPath)
+
+		return pm.ToString()
+	}
+
+	return pc.ToMap().ToString()
+}
+
+// Merge will return the merged ProcConf, the values in the input will take precedent, does not mutate the original.
```

Comment:
```suggestion
// Merge returns the merged ProcConf, the values in the input will take precedent, does not mutate the original.
```
above you just used present tense, so I assume we can do here the same.